talent/hiring_process.md

```diff
@@ -0,0 +1,78 @@
+# Hiring Process Explained
+
+This document explains the internal hiring process at source{d} and the responsibilities for each person related to it.
+
+
+### Job Description
+
+- Job Description is written by the lead engineers with the Talent's Manager support. 
+
+- For every meeting between the lead and Anna there will be a PR in the [minutes](https://github.com/src-d/minutes) repo and the job descriptions will be uploaded at the private repository https://github.com/src-d/company.
+
+- CTO and CEO should be added as reviewers (but should not be bottlenecks to continue).
+
+- Anna posts the job description through Lever on the source{d} website.
+
+### Sourcing
+
+- Lead engineers are responsible to identity people to be contacted and other sourcing channels such as niche job boards. 
+
+- At any given moment the Talent Manager is able to drop in and work on finding more developers.
+
+- Developer lists/sourcing channels should continuously be defined and refreshed after communication between the Talent Manager and the leads. 
+
+- The Talent Manager is responsible to work with the Developer Community team in order to gain people through community actions (Twitter/Blog etc.). 
+
+### First Contact
+
+- Any developer should be contacted by the lead engineers or under their name at least (after together defining the outgoing message). In case the Talent Manager is contacting them, it should be done from the lead's email acccount. It's a motivating factor for the developers to be contacted by the lead engineers. 
+
+### Screening
+
+- If the developer is an applicant, they should be accepted/rejected by the lead engineers during the screening phase. 
```

Comment:
Any applicant should have an interview with the lead? maybe we can limit this depending of the source of the applicant. If we know that the source is worth it, then for me is ok to do the first interview, but if the applicant can be anyone, then I would prefer to apply a filter first.

Reply:
I think it is a good idea to have the lead doing a first screening, to present the company, the job, talk with the candidate about their past and find any potential red flag.

Reply:
I'm not sure how debatable this point is, but personally I agree \w @ajnavarro - it could be lead by default, but in some cases should be possible to be handled by Talent Manager as well.

Reply:
@tsolakoua will ask the leads to review every application in Lever (so no calls needed here), that isn't an obvious no (still quite often we get applications where there is an obvious mismatch between the requirements and the person). If this person's application looks promising we expect the leads to do the first call for exactly the reasons @rporres mentions. 

Know that for the vast majority of first calls you're doing they are coming from the original list of people you've identified, so in the end, this shouldn't be a big burden. If it becomes one, we can revisit this topic.